MaterialLib/MPL/Properties/PermeabilityMohrCoulombFailureIndexModel.h

```diff
@@ -0,0 +1,127 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * Created on June 4, 2020, 10:13 AM
+ */
+
+#pragma once
+
+#include "MaterialLib/MPL/Property.h"
+#include "MaterialLib/MPL/VariableType.h"
+
+namespace ParameterLib
+{
+struct CoordinateSystem;
+template <typename T>
+struct Parameter;
+}  // namespace ParameterLib
+
+namespace MaterialPropertyLib
+{
+/**
+ * \brief A failure index dependent permeability model \cite wangetalPerm2020
+ *
+ * \f[ \mathbf{k} =
+ *      \mathbf{k}_0+ H(f-1) k_\text{r} \mathrm{e}^{b f}\mathbf{I}\f]
+ *
+ * where
+ *   \f$\mathbf{k}_0\f$ is the intrinsic permeability.
+ *  of the undamaged material,
+ * \f$H\f$ is the Heaviside step function, \f$f\f$ is the failure index,
+ * \f$k_\text{r}\f$ is a reference permeability,
+ * \f$b\f$ is a fitting parameter.
+ * \f$k_\text{r}\f$ and \f$b\f$ can be calibrated by experimental data.
+ *
+ * The failure index \f$f\f$ is calculated from  the Mohr Coulomb failure
+ * criterion comparing an acting shear stress for the shear dominated failure.
+ * The tensile failure is governed by an input parameter of
+ * tensile_strength_parameter .
+ *
+ *  The Mohr Coulomb failure
+ * criterion \cite labuz2012mohr takes
+ * the form
+ *   \f[\tau(\sigma)=c-\sigma \mathrm{tan} \phi\f]
```

Comment:
Other documentation says sigma_n. Writing only sigma looks like full stress tensor.

Reply:
They are the commonly use stress symbols for the MC model. I will copy the context to other doc. Normally in the mechanics notations, the stress tensor symbols take the bold type.

Reply:
That's fine. Choose one consistent notation for the pdf, doxygen, and the code. Makes life and error hunting easier.